[android] Fix padding addition

mfazekas opened the pull request:
Fixes: #956

mfazekas commented:
Testcase 

```js

import React from 'react';
import {SafeAreaView, Button} from 'react-native';
import MapboxGL, {
  MapView,
  ShapeSource,
  LineLayer,
  Camera,
} from '@react-native-mapbox-gl/maps';

import accessToken from './accesstoken.json';

MapboxGL.setAccessToken(accessToken);

const aLine = {
  type: 'LineString',
  coordinates: [[-74.00597, 40.71427], [-74.00697, 40.71527]],
};

class BugReportExample extends React.Component {
  render() {
    return (
      <SafeAreaView style={{flex: 1, flexDirection: 'column'}}>
        <Button
          title="fitBounds"
          onPress={() => {
            this.camera.fitBounds(
              [-74.00597, 40.71427],
              [-74.00697, 40.71527],
              [0, 0, 300, 0],
            );
          }}
        />
        <Button
          title="fitBounds2"
          onPress={() => {
            this.camera.fitBounds(
              [-74.00597, 40.71427],
              [-74.00697, 40.71527],
              [300, 0, 0, 0],
            );
          }}
        />
        <MapView style={{flex: 1}}>
          <Camera
            centerCoordinate={[-74.00597, 40.71427]}
            zoomLevel={14}
            ref={ref => {
              this.camera = ref;
            }}
          />
          <ShapeSource id="idStreetLayer" shape={aLine}>
            <LineLayer id="idStreetLayer" />
          </ShapeSource>
        </MapView>
      </SafeAreaView>
    );
  }
}

export default BugReportExample;

```

ferdicus commented on example/package.json:
```diff
@@ -14,6 +14,7 @@
     "@mapbox/geo-viewport": "^0.4.0",
     "@mapbox/mapbox-sdk": "^0.6.0",
     "@react-native-community/masked-view": "^0.1.7",
+    "@react-native-community/toolbar-android": "^0.1.0-rc.2",
```
why this?

mfazekas replied:
@ferdicus when running example it was complaining about missing `toolbar-android`, will remove and retry.

sunny-lirr commented:
This reverts #202. Has the iOS behavior changed to make that unnecessary? It seems almost like we fix one or the other, since they're opposites.

mfazekas commented:
@sunny-lirr 

> This essentially reverts #202. Has the iOS behavior changed to make that unnecessary? It seems almost like we fix one or the other, since they're opposites.

But since #202 

1.  there were [changes](https://github.com/mapbox/mapbox-gl-native/pull/16067) in how mapbox adds camera padding
2. that caused #841 
3. which was fixed by #941 
4. But that fix caused #956.
5. And this PR attempts to fix #956.

I'll try to see how this PR works with issue mentioned in #201. 

Padding on iOS is currently broken see #841 and i don't see an easy way to work around that - see [underlying issue.](https://github.com/mapbox/mapbox-gl-native-ios/issues/198)

mfazekas commented:
@sunny-lirr
i've changed the PR so that contentInset is now added to padding. (But unlinke #202 we don't add previous padding but only content inset)
